src/System.Private.CoreLib/shared/System/Number.Dragon4.cs

```diff
@@ -12,6 +12,11 @@ internal static partial class Number
     {
         private static unsafe void Dragon4(double value, int precision, ref NumberBuffer number)
         {
+            const double Log10V2 = 0.30102999566398119521373889472449;
```

Comment:
These were moved out of the `NumberBuffer.cs` file because they are only used by `Dragon4`